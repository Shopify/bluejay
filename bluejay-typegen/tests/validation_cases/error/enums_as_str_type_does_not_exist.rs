#[bluejay_typegen::typegen([
    type Query {
        foo: String
    }
], enums_as_str = ["Foo"])]
mod schema {}

fn main() {}
