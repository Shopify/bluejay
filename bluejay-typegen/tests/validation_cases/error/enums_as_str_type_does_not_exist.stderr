error: No enum definition named Foo
 --> tests/validation_cases/error/enums_as_str_type_does_not_exist.rs:5:20
  |
5 | ], enums_as_str = ["Foo"])]
  |                    ^^^^^
